### ISSUE TITLE
check-script-ref-and-source doesn't consider seed files

### ISSUE THREAD

**Issue by petlofg** [Montreal-Analytics/dbt-gloss#264]
**Describe the bug**
The check-script-ref-and-source hook does not seem to treat refs that reference seeds as valid references. The hook fails if any such reference exists in the dbt model. The failure message is: 
Missing model (ref) model.test.seeded_test_file

**To Reproduce**
1. Add a new new seed file to the dbt project.
2. Add a new model file that references the seed file.
3. Stage the changes.
4. Run command: pre-commit run check-script-ref-and-source

Seed file seeded_test_file.csv:
```csv
COL1,COL2,COL3
A,B,C
D,E,F
```

Model file test.sql:
```sql
SELECT COL1, COL2, COL3
FROM {{ ref("seeded_test_file") }}
```

**Expected behavior**
The hook should not report failure.

**Version:**
v2.0.4

**Additional context**
In the check_refs_sources function in the check_script_ref_and_source.py file, the ref_id is set to start with "model.". Seed files start with "seed." in the manifest as far as I can tell. I haven't been able to verify that this is the issue, but it might be related.

**Comment by ric-sylvera** [comment]
Same for snapshots. It is looking for `model.<<project name>>.<<snapshot name>>` in the manifest nodes, but there the id is `snapshot.<<project name>>.<<snapshot name>>`.

This is after bumping to the latest v2.0.4 and moving it back down to v1.2.1 fixed the issue.

**Comment by robinreeves** [comment]
I see the same issue with seeds and snapshots. It appears the change in behavior occurred in version 2.0.4. If I drop the version down to 2.0.3, no issues.

**Comment by BAntonellini** [comment]
Hello everyone.

As pointed above, this issue was fixed with PR #266 

When we added support for versioned refs, we unintentionally limited `ref()` discovery to Models only.

Feel free to try using `v2.0.5` and let us know. Thanks!

**Comment by petlofg** [comment]
Thank you for the fast response to this issue! I have tested it in v2.0.5 and it works fine for my use case.
